src/applications/facility-locator/components/search-results-items/VaFacilityResult.jsx

```diff
@@ -36,6 +41,7 @@ const VaFacilityResult = ({ location, query, index }) => {
         </span>
         <LocationAddress location={location} />
         <LocationDirectionsLink location={location} from={'SearchResult'} />
+        {isCovid19Search && <Covid19Alert />}
```

Comment:
Can we add a test to src/applications/facility-locator/tests/components/VaFacilityResult.unit.spec.jsx to cover this logic?